sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -325,6 +325,18 @@ public partial class SimpleFieldAttribute : System.Attribute
 }
 namespace Azure.Search.Documents.Indexes.Models
 {
+    public partial class AmlSkill : Azure.Search.Documents.Indexes.Models.SearchIndexerSkill
+    {
+        public AmlSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs, string resourceId) { }
+        public AmlSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs, System.Uri scoringUri) { }
+        public AmlSkill(System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IEnumerable<Azure.Search.Documents.Indexes.Models.OutputFieldMappingEntry> outputs, System.Uri scoringUri, string authenticationKey) { }
+        public string AuthenticationKey { get { throw null; } set { } }
```

Comment:
Should this be exposed?  Or should we treat this like a credential that can be rolled?  We had the same conversation per https://apiview.dev/Assemblies/Review/f291bdd4488149748454e2e2fa18a821#Azure.Search.Documents.Indexes.Models.KnowledgeStore and I'd either copy that if it GA'ed or we can talk about fixing both to be more like credentials.

Reply:
`KnowledgeStore` [has GAed](https://github.com/Azure/azure-sdk-for-net/blob/0ebd32c0359bcd7a20b1708683c323a0b26b618a/sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs#L890) and it has getter and setter on its `StorageConnectionString`.